### PR TITLE
Add cumulativeEarningsWei to Operator and OperatorDailyBucket entities

### PR DESCRIPTION
**Pull request by samigabor**

### CODE REVIEW

File: packages/network-subgraphs/src/helpers.ts [hunk 1]
```diff
@@ -157,6 +158,7 @@ export function loadOrCreateOperatorDailyBucket(contractAddress: string, timesta
         bucket.profitsWei = BigInt.zero()
         bucket.lossesWei = BigInt.zero()
         bucket.operatorsCutWei = BigInt.zero()
+        bucket.cumulativeEarningsWei = operator.cumulativeEarningsWei
```
Comment (jtakalai) [inline packages/network-subgraphs/src/helpers.ts:161]: please move to line 153

**Review by jtakalai — APPROVED** [review]

File: packages/network-subgraphs/src/operator.ts [hunk 2]
```diff
@@ -138,11 +138,13 @@ export function handleProfit(event: Profit): void {
     let operator = loadOrCreateOperator(operatorContractAddress)
     operator.cumulativeProfitsWei = operator.cumulativeProfitsWei.plus(valueIncreaseWei)
     operator.cumulativeOperatorsCutWei = operator.cumulativeOperatorsCutWei.plus(operatorsCutDataWei)
+    operator.cumulativeEarningsWei = operator.cumulativeProfitsWei.plus(operator.cumulativeOperatorsCutWei)
     operator.save()
 
     let bucket = loadOrCreateOperatorDailyBucket(operatorContractAddress, event.block.timestamp)
     bucket.profitsWei = bucket.profitsWei.plus(valueIncreaseWei)
     bucket.operatorsCutWei = bucket.operatorsCutWei.plus(operatorsCutDataWei)
+    bucket.cumulativeEarningsWei = bucket.profitsWei.plus(bucket.operatorsCutWei)
```
Comment (jtakalai) [inline packages/network-subgraphs/src/operator.ts:147]:
```suggestion
```

Comment (jtakalai) [reply packages/network-subgraphs/src/operator.ts:147]: Another suggestion:
```suggestion
    bucket.cumulativeEarningsWei = operator.cumulativeEarningsWei
```
with this, maybe do the update for the other "absolute" values as well

File: packages/network-subgraphs/src/helpers.ts (hunk 1 above)
Comment (jtakalai) [inline packages/network-subgraphs/src/helpers.ts:161]: this should be together with absolute values

Comment (jtakalai) [reply packages/network-subgraphs/src/helpers.ts:161]: note: delegatorCountAtStart is actually a third category: legitimately "start of bucket" type of values.

**Review by jtakalai — CHANGES_REQUESTED** [review]
`cumulativeEarningsWei` should work like the other "absolute" values in the bucket

File: packages/network-subgraphs/src/helpers.ts (hunk 1 above)
Comment (samigabor) [reply packages/network-subgraphs/src/helpers.ts:161]: moved

**Review by samt1803 — APPROVED** [review]

File: packages/network-subgraphs/schema.graphql [hunk 3]
```diff
@@ -321,6 +324,9 @@ type OperatorDailyBucket @entity {
 
   "Sum of operator's share of earnings today"
   operatorsCutWei: BigInt!
+
+  "Sum of earnings today, including operator's share"
```
Comment (jtakalai) [inline packages/network-subgraphs/schema.graphql:328]:
```suggestion
  "Sum of the operator's lifetime earnings (including operator's cut)"
```

**Review by jtakalai — CHANGES_REQUESTED** [review]
update comment, then good to merge 👍

**Pull request merged**